atlasclient/client.py

```diff
@@ -72,10 +72,17 @@ def __init__(self, host, port=None, username=None, password=None,
         if identifier is None:
             identifier = 'python-atlasclient'
 
-        self.client = HttpClient(host=self.base_url, username=username,
-                                 password=password, identifier=identifier,
-                                 validate_ssl=validate_ssl, timeout=timeout,
-                                 max_retries=max_retries, auth=auth)
+        if username is not None and password is not None and oidc_token is None:
```

Comment:
@pengfei99 I believe you don't need to make this check here. simply pass all the arguments to `HttpClient` in this case.